spec/requests/users_controller_spec.rb

```diff
@@ -3688,6 +3688,31 @@ def enabled?
       expect(response.status).to eq(200)
       expect(response.parsed_body["cannot_see"][user1.username]).to eq("muted_topic")
     end
+
+    it "returns user who was not invited to topic" do
+      sign_in(Fabricate(:admin))
+
+      get "/u/is_local_username.json", params: {
+        usernames: [group.name], topic_id: private_topic.id
+      }
+
+      expect(response.status).to eq(200)
+      expect(response.parsed_body["valid_groups"]).to include(group.name)
+      expect(response.parsed_body["cannot_see_groups"][group.name]).to eq("not_allowed")
+    end
+
+    it "returns user who was not invited to topic" do
```

Comment:
The description here duplicates the description on line 3692 so I can't really tell what we are trying to test for here.